Python/Product/Common/Strings.resx

```diff
@@ -2876,4 +2876,25 @@ Please specify at least one package.</value>
   <data name="CondaEnvNotDetected" xml:space="preserve">
     <value>Conda environment not detected</value>
   </data>
+  <data name="PyTestIgnoreInfoBarAction" xml:space="preserve">
+    <value>Don't show this message again</value>
+  </data>
+  <data name="PyTestEnableInfoBarAction" xml:space="preserve">
+    <value>Enable pytest</value>
+  </data>
+  <data name="PyTestInstalledConfigurationFileFound" xml:space="preserve">
+    <value>A pytest configuration file was detected in the {0} {1}, but pytest is not enabled.</value>
```

Comment:
when we have replacement tags ie { } we usually put a comment for localization to understand what each one of them is for.
look for examples in this file, I think it's a `<comment>` or `<description>` or something like that.

Reply:
Applies to 2 more strings below